KarhooUISDK/API/KarhooUI.swift

```diff
@@ -13,6 +13,7 @@ public final class KarhooUI {
     public static var colors: KarhooColors = DefaultKarhooColors()
     public static var sideMenuHandler: SideMenuHandler?
     public static var components = KarhooComponents.shared
+    private static var fetureFlagsUpdater = FeatureFlagsUpdater(currentSdkVersion: "1.12.0")
```

Comment:
Could you please use the bundle short version? It'd be nice to avoid another value to update during a release process.